CSLabs.Api/Proxmox/ProxmoxApi.cs

```diff
@@ -18,11 +19,13 @@ public class ProxmoxApi
         private DateTime _loggedInAt = DateTime.MinValue;
         private string _password;
         public HypervisorNode HypervisorNode { get;}
-        public ProxmoxApi(HypervisorNode hypervisorNode, string password)
+        private DefaultContext _context;
+        public ProxmoxApi(HypervisorNode hypervisorNode, string password, DefaultContext context)
         {
             client = new PveClient(hypervisorNode.Hypervisor.Host);
             HypervisorNode = hypervisorNode;
             _password = password;
+            _context = context;
```

Comment:
@jasekiw Is this the right way to add the database context? The ProxmoxManager is what creates a new ProxmoxApi and it has a context so it can pass it into the constructor when it creates it. I want to double check because it also broke some test case stuff, in one of the test files where it created a new ProxmoxApi